tools/roadmap/validate_dag.py

```
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+import json
+import os
+import re
+import subprocess
+import sys
+from typing import Sequence, Tuple, List, Dict
```

Ruff (0.14.3)

Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)